scripts/remove.sh

```diff
@@ -302,3 +302,29 @@ remove_nginx(){
     ERROR_MSG=" Looks like Nginx was already removed!\n Skipping..."
   fi
 }
+
+remove_klipperscreen(){
+  data_arr=(
+  $KLIPPERSCREEN_DIR
+  $KLIPPERSCREEN_ENV_DIR
+  /etc/systemd/system/KlipperScreen.service
+  )
+  print_error "KlipperScreen" && data_count=()
+  if [ "$ERROR_MSG" = "" ]; then
+    #remove KlipperScreen dir
+    if [ -d $KLIPPERSCREEN_DIR ]; then
+      status_msg "Removing KlipperScreen directory ..."
+      rm -rf $KLIPPERSCREEN_DIR && ok_msg "Directory removed!"
+    fi
+    if [ -d $KLIPPERSCREEN_ENV_DIR ]; then
+      status_msg "Removing KlipperScreen VENV directory ..."
+      rm -rf $KLIPPERSCREEN_ENV_DIR && ok_msg "Directory removed!"
+    fi
+    #remove KlipperScreen systemd file
+    if [ -e /etc/nginx/sites-available/mainsail ]; then
+      status_msg "Removing KlipperScreen configuration for Nginx ..."
```

Comment:
The if condition here looks like a typo.
You probably wanted to check for `[ -e /etc/systemd/system/KlipperScreen.service ]`.

And the status_msg does not suit the action followed by it.
You probably want to call it `status_msg "Removing KlipperScreen Service ..."`